### ISSUE TITLE
Second pass of tuning xgemm :  5e7 GEMMs to consider ?

### ISSUE THREAD

**Issue by newling** [CNugteren/CLBlast#187]
I am tuning sgemm. `./clblast_tuner_xgemm`
The first "run" (with fraction = 1.0) considers 488 GEMMs.
Then a second "run" starts (with fraction = 512.0) which looks like it's going to consider 57819136 GEMMs.
I stopped at i=1439, this would take forever!

```
.
.
.
[ RUN      ] Running Xgemm
[       OK ] Completed Xgemm (1.0 ms) - 1439 out of 57819136
```

**Comment by CNugteren** [comment]
You are right to stop that :-) It turned out that PR #173 wasn't properly tested and it wouldn't take the inverse of the fraction argument (so running 512 times the whole kernel set instead of 1/512th of all kernels). That's fixed now in commit 6194d43, sorry for the inconvenience.

**Comment by newling** [comment]
It's working now, thanks for the quick fix!
